docs/release-notes/index.md

```diff
@@ -27,7 +27,7 @@ To check for security updates, go to [Security announcements for the Elastic sta
 
 ### Features and enhancements [9-0-0-features-enhancements]
 
-* **Tail-based sampling**: Storage layer is rewritten to use Pebble database instead of BadgerDB. The new implementation offers a substantial throughput increase while consuming significantly less memory. Disk usage is lower and more stable. See PR for benchmark details. ([#15235](https://github.com/elastic/apm-server/pull/15235))
+* **Tail-based sampling**: Storage layer is rewritten to use Pebble database instead of BadgerDB. The new implementation offers a substantial throughput increase while consuming significantly less memory. Disk usage is lower and more stable. See APM [Transaction sampling](../../solutions/observability/apps/transaction-sampling) docs for benchmark details. ([#15235](https://github.com/elastic/apm-server/pull/15235))
```

Comment:
@colleenmcginnis this will be broken in the PR review and CI check, but I assume it'll result in the correct link in the docs. Do you have a better alternative to achieve the same?

Reply:
You can use a [cross-repo link](https://elastic.github.io/docs-builder/syntax/links/#cross-repository-links). This should make the CI check pass.

```suggestion
* **Tail-based sampling**: Storage layer is rewritten to use Pebble database instead of BadgerDB. The new implementation offers a substantial throughput increase while consuming significantly less memory. Disk usage is lower and more stable. See APM [Transaction sampling](docs-content://solutions/observability/apps/transaction-sampling.md) docs for benchmark details. ([#15235](https://github.com/elastic/apm-server/pull/15235))
```

Reply:
TIL!